### PR TITLE
feat: Use XControls in NextQueries and Layout

### PR DESCRIPTION
**Pull request by annacv**
# Pull request template
 
Use XControls to config:
- The `maxNextQueriesPerGroup` to be shown
- Column options to display for desktop

## Motivation and context

- [ ] Dependencies. If any, specify:
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

**Test with `dress` and `useXControls` composable inside `custom-query-preview.vue` component to quickly see the changes** ([you can check this commit to do so](https://github.com/empathyco/x-archetype/pull/360/commits/9e8cd12e4f9ae654ebd46401bfdaac28acf2d512))

Tests performed according to [testing guidelines](https://github.com/empathyco/x/blob/main/.github/contributing/tests.md): 

## Checklist:

- [ ] My code follows the **[style guidelines](https://github.com/empathyco/x/blob/main/.github/CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

**Comment by support-empathy** [comment]
Check **PR 360** preview 👀 <br> <br> [https://x.test.empathy.co/preview/360/index.html](https://x.test.empathy.co/preview/360/index.html)

**Comment by support-empathy** [comment]
Check **PR 360** preview 👀 <br> <br> [https://x.test.empathy.co/preview/360/index.html](https://x.test.empathy.co/preview/360/index.html)

**Comment by support-empathy** [comment]
Check **PR 360** preview 👀 <br> <br> [https://x.test.empathy.co/preview/360/index.html](https://x.test.empathy.co/preview/360/index.html)

File: src/composables/use-store.composable.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { RootXStoreState, StatusState, XModuleName, XModulesTree } from '@empathyco/x-components';
+
+/**
+ * Provides access to the global store object or to the specific store module if it's provided.
+ *
+ * @param storeModule - The {@link XModuleName}.
+ * @returns A state object.
+ */
+export function useStore(storeModule?: keyof XModulesTree): RootXStoreState | StatusState {
```
Comment (CachedaCodes) [inline src/composables/use-store.composable.ts:11]: Add a TODO to change when we add the useStore composable in x

**Comment by support-empathy** [comment]
Check **PR 360** preview 👀 <br> <br> [https://x.test.empathy.co/preview/360/index.html](https://x.test.empathy.co/preview/360/index.html)

**Review by CachedaCodes — APPROVED** [review]

**Comment by support-empathy** [comment]
Check **PR 360** preview 👀 <br> <br> [https://x.test.empathy.co/preview/360/index.html](https://x.test.empathy.co/preview/360/index.html)

**Pull request merged**